src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -267,6 +264,8 @@ public void TestStartOnUnixWithBadFormat()
         [DllImport("libc")]
         private static extern int chmod(string path, int mode);
 
+        private List<string> GetAllowedProgramsToRun() => new List<string> { "xdg-open", "gnome-open", "kfmclient" };
```

Comment:
an array is more efficient (compact) when you have a fixed set of items. 
also you can mark this readonly.

Reply:
I mean, if you make it a field.

Reply:
eh, you're using Any() .. fine.